airgun/views/flatpak.py

```diff
@@ -1,13 +1,19 @@
 from widgetastic.widget import Text
 from widgetastic_patternfly5 import (
+    Button as PF5Button,
     Menu as PF5Menu,
+    Modal as PF5Modal,
     Pagination,
 )
 from widgetastic_patternfly5.ouia import (
     PatternflyTable as PF5OUIATable,
+    Text as OUIAText,
```

Comment:
I would use this naming scheme, we already use elsewhere
```suggestion
    Text as PF5OUIAText,
```